modules/tile-converter/docs/api-reference/i3s-converter.md

```diff
@@ -41,4 +41,4 @@ Converts a tileset to I3S format
 - `options.egmFilePath` location of \*.pgm file to convert heights from ellipsoidal to gravity-related format. A model file can be loaded from GeographicLib https://geographiclib.sourceforge.io/html/geoid.html
 - `options.token` ION token of input tileset
 - `options.draco` Whether the converter create DRACO compressed geometry in path "layers/0/nodes/xxx/geometries/1" along with non-compressed geometry in path "layers/0/nodes/xxx/geometries/0"
-- `options.validateBoundingVolumes` Enable/Disable Bounding Volumes Validation (Check if child bounding volume inside parent bounding volume)
+- `options.validate` Enable/Disable Validation while conversion
```

Comment:
```suggestion
- `options.validate` Enable validation
```

- Add a section describing validation

```
### Validation

By specifying the --validate parameter, the tile-convert will perform checks on the tileset data. The following checks are performed:
- Bounding volume validation - Bounding boxes are checked against...

Also

- Can the tile-converter be run with just `--validate` (without actually converting)

Reply:
Fixed. Yes ideally we should use `--validate` flag independently of conversion process. It doesn't looks like small change and it is big risk to break current converter work. We should do it in separate milestone.